.gitattributes

```diff
@@ -3,3 +3,9 @@
 test/**/*.js -crlf
 test/es6/HTMLComments.js binary diff=cpp
 *.wasm binary
+*.cpp text eol=lf diff=cpp
+*.h text eol=lf diff=cpp
+*.inl text eol=lf diff=cpp
+*.vcproj text eol=crlf diff=xml
+*.vcxproj text eol=crlf diff=xml
```

Comment:
so far, AFAIK, we've been treating these files as LF with no ill effects.

Reply:
+1, where did this change originate from?

Reply:
Sounds like @Penguinwizzard recommended this spec as a way to help git deal with merge conflicts on line endings -- and it possibly had the opposite from the intended effect. :P

Reply:
There's an easy way to strip line ending changes from a PR. I'll have to dig it up from email somewhere.